core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -433,10 +432,21 @@ import scala.compat.java8.FutureConverters._
       private def scanForDelayedEvents(): Unit = {
         if (!stageState.delayedScanInProgress) {
           updateStageState(_.copy(delayedScanInProgress = true))
-          val startOfScan = settings.eventsByTagBacktrack.period match {
-            case Max             => Uuids.startOf(stageState.currentTimeBucket.previous(1).key)
-            case Fixed(duration) => Uuids.startOf((Uuids.unixTimestamp(stageState.fromOffset) - duration.toMillis))
-          }
+
+          val startOfPreviousBucket = stageState.currentTimeBucket.previous(1).key
+          val currentTime = Uuids.unixTimestamp(stageState.fromOffset)
```

Comment:
`currentTime` but it's `fromOffset`?

Reply:
Confusing name perhaps, that is the currentTime of this query. I'll rename it to fromOffsetTime